src/components/List/drop-down-list.scss

```diff
@@ -0,0 +1,92 @@
+@mixin icon {
+    position: absolute;
+    top: 33%;
+    left: 70%;
+    color: #6a687e;
+    cursor: pointer;
+    height: 20px;
+    width: 20px;
+  }
+  @mixin dropitem {
+    color: black;
+    padding: 12px 16px;
+    cursor: pointer;
+    font-family: Verdana;
+    font-size: 20px;
+    text-align: left;
+  }
+  .container {
+   width: 100%;
+   height:800px;
+  }
+
+  .inputContainer {
+    padding: 5px;
+    align-items: center;
```

Comment:
you are using `align-items` property but I don't see the use of flex

Reply:
i was trying sth and forgot to to remove it